pages/product/[productId]/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { Panel, Text } from "@bigcommerce/big-design";
+import { useRouter } from "next/router";
+import ErrorMessage from "@components/error";
+import Loading from "@components/loading";
+import { useProductInfo, useProductList } from "@lib/hooks";
+
+const SidePanel = () => {
```

Comment:
`SidePanel` is a bit ambiguous - we might want to call this `ProductAppExtension`